tests/tdl/test-parser-ast.cpp

```diff
@@ -1,17 +1,35 @@
 // NOLINTBEGIN(cert-err58-cpp,cppcoreguidelines-avoid-do-while,readability-function-cognitive-complexity,cppcoreguidelines-avoid-non-const-global-variables,cppcoreguidelines-avoid-c-arrays,modernize-avoid-c-arrays)
 
+#include <string>
+#include <string_view>
+
 #include <catch2/catch_test_macros.hpp>
 #include <ystdlib/error_handling/Result.hpp>
 
 #include <spider/tdl/parser/ast/Node.hpp>
+#include <spider/tdl/parser/ast/node_impl/Identifier.hpp>
 
 namespace {
 TEST_CASE("test-ast-node", "[tdl][ast][Node]") {
     using spider::tdl::parser::ast::Node;
+    using spider::tdl::parser::ast::node_impl::Identifier;
     using ystdlib::error_handling::Result;
 
-    Result<void> const result{Node::ErrorCode{Node::ErrorCodeEnum::ChildIndexOutOfBounds}};
-    REQUIRE(result.has_error());
+    SECTION("Identifier") {
+        constexpr std::string_view cTestName{"test_name"};
+        constexpr std::string_view cSerializedIdentifier{"[Identifier]: test_name"};
+
+        auto const node{Identifier::create(std::string{cTestName})};
+        auto const* identifier{dynamic_cast<Identifier const*>(node.get())};
+        REQUIRE(nullptr != identifier);
+
+        REQUIRE(nullptr == identifier->get_parent());
+        REQUIRE(identifier->get_name() == cTestName);
```

Comment:
According to code style guide, constants should be put before `==`.
```suggestion
        REQUIRE(cTestName == identifier->get_name());
```

Reply:
As discussed offline, rvalue has priority to be put on the lhs of `==`.